src/vs/gitpod/browser/workbench/workbench.ts

```diff
@@ -825,6 +826,9 @@ async function doStart(): Promise<IDisposable> {
 		urlCallbackProvider: new PollingURLCallbackProvider(),
 		credentialsProvider,
 		productConfiguration: {
+			nameShort: product.nameShort + (info.ideAlias === 'code-latest' ? ' - Insiders' : ''),
+			nameLong: product.nameLong + (info.ideAlias === 'code-latest' ? ' - Insiders' : ''),
+			version: product.version + (info.ideAlias === 'code-latest' ? '-insider' : ''),
```

Comment:
Maybe there should be an 's' in the version tag as well?

Reply:
that's all right, seems to be aligned with how VS Code Desktop handles it